packages/zowe-explorer-ftp-extension/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Change Log
 
+## `1.16.0`
+
+- Added the function that closes a connection
```

Comment:
This is pretty vague for me. What about: "Added the function that closes the connection at the end of each FTP operation" ?